doc/community-build/docker/cortx-all/README.md

```diff
@@ -46,11 +57,11 @@ This document provides step-by-step instructions to build CORTX binaries and con
    docker run --rm -v /var/artifacts:/var/artifacts -v /mnt/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:rockylinux-8.4 make clean cortx-all-rockylinux-image
    ```
    
-   **Note:** This process takes some time to complete building the CORTX packages during `/var/artifacts/0 /` implementation phase.
+   **Note:** This process takes around 5Hrs to build entire CORTX packages with minimum VM specifications. This time will vary based on VM specification and Network Speed. 
```

Comment:
rephrase as,

**Note:** This process takes around 5Hrs to build entire CORTX packages with minimum VM specifications where time can vary based on VM specification and network speed.

Reply:
Fixed please review